portfolio/src/main/java/com/google/sps/servlets/HomeServlet.java

```diff
@@ -45,13 +44,18 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
        
       response.getWriter().println("<p>Logout <a href=\"" + logoutUrl + "\">here</a>.</p>");
     } else {
-      String urlToRedirectToAfterUserLogsIn = "/";
-      String loginUrl = userService.createLoginURL(urlToRedirectToAfterUserLogsIn);
-
-      response.getWriter().println("<p>Hello stranger.</p>");
-      
-     
-      response.getWriter().println("<p>Login <a href=\"" + loginUrl + "\">here</a>.</p>");
-    }
+       String urlToRedirectToAfterUserLogsIn = "/";
+       String loginUrl = userService.createLoginURL(urlToRedirectToAfterUserLogsIn);
+       response.getWriter().println("<p>Hello stranger.</p>");
+       response.getWriter().println("<p>Before make comment, Please login first!</p>");
+       response.getWriter().println("<p>Login <a href=\"" + loginUrl + "\">here</a>.</p>");
+      //response.sendRedirect("/"+loginUrl);
```

Comment:
Consider deleting these commented-out lines. Note that these lines have not been checked into git yet, so if you delete them they will not be visible in your previous commits. That said, I would still remove them - if you think you might need them later, just copypaste them into a saved file somewhere.